protocol/x/clob/types/keys.go

```diff
@@ -18,94 +18,91 @@ const (
 const (
 	// ProcessProposerMatchesEventsKey is the key to retrieve information about how to update
 	// memclob state based on the latest block.
-	ProcessProposerMatchesEventsKey = "process_proposer_matches_events"
+	ProcessProposerMatchesEventsKey = "ProposerEvents"
 
 	// SubaccountLiquidationInfoKeyPrefix is the prefix to retrieve the liquidation information
 	// for a subaccount within the last block.
-	SubaccountLiquidationInfoKeyPrefix = "subaccount_liquidation_info/"
+	SubaccountLiquidationInfoKeyPrefix = "SaLiqInfo/"
 
 	// LiquidationsConfigKey is the key to retrieve the liquidations config.
-	LiquidationsConfigKey = "liquidations_config"
+	LiquidationsConfigKey = "LiqCfg"
 
 	// EquityTierLimitConfigKey is the key to retrieve the equity tier limit configuration.
-	EquityTierLimitConfigKey = "equity_tier_limit_config"
+	EquityTierLimitConfigKey = "EqTierLim"
 
 	// BlockRateLimitConfigKey is the key to retrieve the block rate limit configuration.
-	BlockRateLimitConfigKey = "block_rate_limit_config"
+	BlockRateLimitConfigKey = "RateLim"
```

Comment:
Nit: I'd add `Cfg` here for clarity. The non-prefix keys are less important since they only occur once.

Maybe if you keep state for the past `X` blocks then it's replicated on disk `X` times but doesn't seem too bad for these param/config keys